lib/charms/traefik_k8s/v0/ingress_per_unit.py

```diff
@@ -812,16 +712,16 @@ def _publish_ingress_data(self, host: Optional[str], port: int):
         }
         self.relation.data[self.unit]["data"] = _serialize_data(data)
 
-    def request(self, *, host: str = None, port: int):
-        """Request ingress to this unit.
+    def provide_ingress_requirements(self, *, host: str = None, port: int):
```

Comment:
Why do we have `_provide_ingress_requirements` and `provide_ingress_requirements`? The latter seems to just call the former?